src/job/http.go

```diff
@@ -71,9 +71,14 @@ func singleRequestJob(ctx context.Context, args config.Args, globalConfig *Globa
 		log.Printf("Sent single http request to %v", requestConfig.Path)
 	}
 
-	// dataSize :=
 	http.InitRequest(requestConfig, req)
 
+	// dataSize, _ :=
+	_, err = req.WriteTo(metrics.NopWriter{})
```

Comment:
you can just remove this if you don't need to calculate data size

Reply:
That wasn't the final version. You can take a look at the http job now.